pymeasure/instruments/tektronix/afg3152c.py

```diff
@@ -100,6 +104,24 @@ class AFG3152CChannel(Channel):
         """ Control the amplitude offset. It is always in Volt. (float)""",
     )
 
+    # phase degree should not be a query because it always returns rad
+    phase_deg = Instrument.setting(
+        "phase:adjust %e DEG",
+        """ A floating point set comomand that controls
+          the phase in degree units.""",
```

Comment:
Please use the correct [docstring style](https://pymeasure.readthedocs.io/en/latest/dev/coding_standards.html#docstrings) (different from earlier used style):
```suggestion
        """ Set the phase in degrees (float).""",
```

Also below (but with Control instead of Set)